packages/blocks/src/__internal__/rich-text/virgo/affine-text.ts

```diff
@@ -0,0 +1,73 @@
+import {
```

Comment:
IMO virgo shouldn't know the context of the client (which is affine here). But let's refactor this later in next PR.

Reply:
Here affine use virgo as a **package**.

Reply:
This something about how user use virgo because virgo should support developers customize their components and schema.

Reply:
I agree, but the virgo aims to be an infra editor framework like slate and prosemirror. It's not suitable to create customization for every user inside the package. Instead, we should provide an universal way to make virgo can be customized.

Reply:
I'm not sure if we are on the same channel. I will add the basic use document of virgo later. You can read it and see if the evaluation is reasonable. 🙂

Reply:
Really sorry, I made a mistake here. I thought the file is inside the virgo package. Never mind.